onnx/test/shape_inference_test.py

```diff
@@ -1202,6 +1202,23 @@ def test_loop(self):    # type: () -> None
             [make_tensor_value_info('loop_state_final', TensorProto.FLOAT, None),  # shape may change between iterations
              make_tensor_value_info('loop_output', TensorProto.FLOAT, (None, 3))])  # type: ignore
 
+    def test_constantofshape_with_input_shape(self):  # type: () -> None
+        graph = self._make_graph([],
+            [make_node("Constant", [], ['shape'],
+                       value=make_tensor('shape', TensorProto.INT64, (3,), (3, 4, 5))),
+             make_node("ConstantOfShape", ['shape'], ['y'], value=make_tensor('shape', TensorProto.INT32, (1, ), (2, )))],
```

Comment:
The name of the `value` tensor (attribute) is confusing. Can we rename it to `value` or something else?

Reply:
Fixed.